Makefile

```diff
@@ -88,3 +95,25 @@ tidy:
 	@go mod tidy
 	@go mod verify
 	@echo "Done"
+
+packager-check:
```

Comment:
We can check all tools like following: https://github.com/Xuanwo/storage/blob/master/Makefile#L13-L19